src/types/v1/CSN-Interop-Effective.ts

```diff
@@ -315,9 +315,10 @@ export interface CSNInteropRoot {
    * See https://tour.json-schema.org/content/06-Combining-Subschemas/02-id-and-schema
    *
    */
-  $schema?: string;
+  $schema?: ("https://sap.github.io/csn-interop-specification/spec-v1/csn-interop-effective.schema.json" | string) &
```

Comment:
This is a nice side-effect in TypeScript, that now the official URI to the schema becomes known at type level :)